Tests/EventListener/CsrfListenerTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sensio\Bundle\FrameworkExtraBundle\Tests\EventListener;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Csrf;
+use Sensio\Bundle\FrameworkExtraBundle\EventListener\CsrfListener;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Security\Csrf\CsrfToken;
+
+class CsrfListenerTest extends \PHPUnit_Framework_TestCase
+{
+    protected $listener;
+
+    protected $tokenManager;
+
+    protected function setUp()
+    {
+        if (!interface_exists('Symfony\Component\Security\Csrf\CsrfTokenManagerInterface')) {
+            $this->markTestSkipped('This test requires "symfony/security-csrf".');
+        }
```

Comment:
This should be removed as we list all required deps in composer.json so it should always be present.